exercises/async-activity-completion/solution/workflow.py

```diff
@@ -0,0 +1,42 @@
+import asyncio
+from typing import List
+
+from temporalio import activity, workflow
+from temporalio.client import Client
+from datetime import timedelta
+
+class GreetingComposer:
+    def __init__(self, client: Client) -> None:
+        self.client = client
+
+    @activity.defn
+    async def compose_greeting(self) -> str:
```

Comment:
Should this be `-> List[str]` to match workflow expectation? (may want a type checker on the repo)